package.json

```diff
@@ -13,6 +13,7 @@
     "lint-check-jessie": "eslint -c '.eslintrc-jessie.js' '**/*.{js,jsx}'"
   },
   "devDependencies": {
+    "@agoric/evaluate": "^1.1.0",
```

Comment:
This is a significant regression: `@agoric/evaluate` latest published version is 2.1.3.  Please upgrade to that version and try again.

Reply:
@michaelfig How can this be "a signigicant regression" when all tests are passing and the  `@agoric/make-importer` has never used  `@agoric/evaluate` but relied on this mock:
```js
const evaluate = new Function(`\
with (arguments[1]) {
// console.log('evaluate', arguments[0]);
return eval(arguments[0]);
}`);
```

Reply:
@michaelfig The version of `@agoric/evaluate` in the `@agoric/transform-module` was 1.x until you updated it to 2.1.3 one week ago in this PR https://github.com/Agoric/transform-module/commit/9d9183aca37e71f8418e9e89226b219ab81f0631

I have been using the 1.x for consistency. If you make changes and don't publish your changes with pull requests, it becomes impossible to follow your changes.

Reply:
Sorry, I misunderstood your comment in #33 as saying the problem existed in `@agoric/evaluate`.

Reply:
@michaelfig, the `@agoric/evaluate` 1.x was used in `@agoric/transform-module`. 

This PR is titled "Use @agoric/evaluate in tests for consistency" because the object is to use the same evaluator everywhere. If you updated the version in ` @agoric/transform-module`, then we will use that one.

Currently, we have more evaluators than test failures not consistent and it's a huge burden when assessing failures.